packages/logger/src/transports/storage.ts

```diff
@@ -108,7 +108,14 @@ export class StorageTransport implements ITransport {
         console.error('StorageTransport.initDB:', err);
       });
       this._onUnload();
-      this._pruneLogs();
+      if (global.localStorage) {
+        const key = `${this._tempKey}-prune-time`;
+        const PrunedTime = global.localStorage.getItem(key);
+        if (PrunedTime !== new Date().toLocaleDateString()) {
+          this._pruneLogs();
```

Comment:
how about add some delay for pruneLog to avoid pruneLog when that instance init when app very busy